dbt_project.yml

```diff
@@ -19,6 +19,6 @@ vars:
     apple_search_ads__adapter: false ## placeholder for ad_reporting package
     
 models:
-  apple_search_ads:
-    materialized: table
+  apple_search_ads: ##
+    materialized: table ##
```

Comment:
We should probably add a materialization config for the `intermediate` models to be `ephemeral`. I don't think they need to be views.